<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QDialog" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать запись</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditID">
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName">
      <property name="placeholderText">
       <string>Название сорта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditRoast">
      <property name="placeholderText">
       <string>Степень обжарки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditType">
      <property name="placeholderText">
       <string>Молотый/в зернах</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDescription">
      <property name="placeholderText">
       <string>Описание вкуса</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPrice">
      <property name="placeholderText">
       <string>Цена</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditVolume">
      <property name="placeholderText">
       <string>Объем упаковки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
